metaflow/current.py

```diff
@@ -69,6 +70,31 @@ def is_running_flow(self):
     def flow(self) -> FlowSpec:
         return self._flow
 
+    def task_log_location(self, log_prefix: str, stream: Optional[str] = None) -> str:
+        """
+        This returns the current Task and Run datastore appropriate location
+        to store logs.
+
+        Args:
+            log_prefix (str): The prefix
+            stream (Optional[str], optional): If None, then the
+                When log_prefix == lightning_logs and stream==None:
```

Comment:
This line `When log_prefix == lightning_logs and stream==None:` confuses me and seems like it shouldn't be there or more was meant to be added?

Reply:
fixed! thank you!